From 02645692b6bc933440724cb27366430665966a3b Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Tue, 8 Sep 2020 14:49:35 +0800
Subject: [PATCH] ARM: dts: rv1126-38x38-v10-emmc: add some optional properties

Keep 0dB for adc/dac volume, and using hardware loopback
which is dac_l+adc_r.

Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
Change-Id: Ie226791c5bd62324552237d98cc10b6bf31ac5b5
---
 arch/arm/boot/dts/rv1126-38x38-v10-emmc.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/rv1126-38x38-v10-emmc.dts b/arch/arm/boot/dts/rv1126-38x38-v10-emmc.dts
index 917582fcf2f9..185adc6b07c4 100644
--- a/arch/arm/boot/dts/rv1126-38x38-v10-emmc.dts
+++ b/arch/arm/boot/dts/rv1126-38x38-v10-emmc.dts
@@ -266,6 +266,9 @@
 		reg = <0x18>;
 		clocks = <&cru MCLK_I2S0_TX_OUT2IO>;
 		clock-names = "mclk";
+		adc-volume = <0xbf>;  /* 0dB */
+		dac-volume = <0xbf>;  /* 0dB */
+		aec-mode = "dac left, adc right";
 		pinctrl-names = "default";
 		pinctrl-0 = <&i2s0m0_mclk &spk_ctl>;
 		assigned-clocks = <&cru MCLK_I2S0_TX_OUT2IO>;
-- 
2.35.3

